codebuild/cd/promote-release.yml

```diff
@@ -12,7 +12,17 @@ phases:
     commands:
       - sudo add-apt-repository ppa:openjdk-r/ppa
       - sudo apt-get update -y
-      - sudo apt-get install openjdk-8-jdk-headless maven -y -f
+      # Android requires 11 for gradle
```

Comment:
any specific reason to not have the android and regular maven publish split?

Reply:
Well, I think they are two different packages, with different package name and built steps. I'd prefer to split them up